app/controllers/api/index.js

```diff
@@ -12,5 +12,17 @@ if(true){
     console.log("hello")
   }
 }
+let j = 1
+if(true){
+  if(j == 1){
+    console.log("hello")
+  }
+}
+let k = 1
+if(true){
+  if(k == 1){
```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/imran-easyeat/sonar-testing/pullRequest?prid=9010495)
```suggestion
  if (k == 1){
```